L1Trigger/L1TMuonEndCap/test/tools/MakeCoordLUT.cc

```diff
@@ -27,7 +27,6 @@
 #include "Geometry/CSCGeometry/interface/CSCLayerGeometry.h"
 #include "Geometry/Records/interface/MuonGeometryRecord.h"
 
-//#include "L1Trigger/CSCCommonTrigger/interface/CSCTriggerGeometry.h"
```

Comment:
Leave file as is, or remove all commented-out lines.

Reply:
The line that was removed was removed by Sven in the CMSSW_10_X_Y codebase. This is because I synchronized my codes to the CMSSW_10_X_Y codebase.